Dockerfile

```diff
@@ -60,8 +59,11 @@ RUN cd /serialbox && \
     /bin/rm -rf /serialbox
 
 # install gt4py
-RUN pip install git+https://github.com/VulcanClimateModeling/gt4py.git@develop && \
-    python -m gt4py.gt_src_manager install
+RUN pip install git+https://github.com/VulcanClimateModeling/gt4py.git@v32 && \
+    python -m gt4py.gt_src_manager install -m 2
+
+# install some python packages
+RUN pip install numpy xarray[complete]
```

Comment:
not required for the PR, but we should probably pin the versions in case they change underneath us in ways that break code. Python packages often aren't too concerned with being backward-compatible.

Reply:
Sounds good, will add to the list of things to think about later